### PR TITLE
Addd align="right" attribute to menu tag

### PR DESCRIPTION
**Pull request by cuu508**
If a dropdown is close to the right edge of the screen, the menu can go off screen. align="right" attribute fixes that. Vanilla bootstrap has something similar, a ".dropdown-menu-right" class.

### CODE REVIEW

**Comment by cognitom** [comment]
Cool! Thanks @cuu508 
I'll merge it asap.

File: src/menu.tag [hunk 1]
```diff
@@ -69,6 +69,11 @@ import { domEvent, syncEvent, parentScope } from 'riot-mixin-pack'
       border-radius: 4px;
       box-shadow: 0 6px 12px rgba(0, 0, 0, .175);
     }
+
+    :scope[align="right"] {
+        left: auto;
+        right: 0;
```
Comment (cognitom) [inline src/menu.tag]: The indentation is supposed to be 2 spaces.

**Comment by cuu508** [comment]
@cognitom good catch, fixed indentation.

**Comment by cognitom** [comment]
@cuu508 Thank you!

**Comment by cognitom** [comment]
`v1.1.0` has been published 😉

**Pull request merged**